OpenRA.Mods.Common/Projectiles/Missile.cs

```diff
@@ -876,8 +884,7 @@ public IEnumerable<IRenderable> Render(WorldRenderer wr)
 						yield return r;
 				}
 
-				var palette = wr.Palette(info.Palette);
-				foreach (var r in anim.Render(pos, palette))
+				foreach (var r in anim.Render(pos, wr.Palette(palette)))
 					yield return r;
```

Comment:
Looking up the palette each render is wasteful because we know it won't change.  Please cache this on first render.